### PR TITLE
WIP: Backend setup

### PR DESCRIPTION
**Pull request by Monal5031**

### CODE REVIEW

File: src/actions/authActions.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export const actionTypes = {
+    SIGN_IN_DONE: 'AUTH.SIGN_IN.DONE',
```
Comment (PiyushPawar17) [inline src/actions/authActions.js]: Keep key-value pair same like SIGN_IN_DONE = 'SIGN_IN_DONE' for all constants.

Comment (piyush-pawar-17) [inline src/actions/authActions.js]: Keep key-value pair same like SIGN_IN_DONE = 'SIGN_IN_DONE' for all constants.

**Review by piyush-pawar-17 — CHANGES_REQUESTED** [review]

**Review by piyush-pawar-17 — APPROVED** [review]

**Comment by anshumanv** [comment]
Deploy fail ka dekh le

**Comment by Monal5031** [comment]
@anshumanv Apologies for late updates done with PR (backend setup now)

PS: Don't look at UI there is no CSS atm

**Review by anshumanv — APPROVED** [review]

**Pull request merged**